zookeeper-client/zookeeper-client-c/README

```diff
@@ -163,3 +163,23 @@ In order to be able to use the zookeeper API in your application you have to
 
 Please take a look at cli.c to understand how to use the two API types. 
 (TODO: some kind of short tutorial would be helpful, I guess)
+
+How to build your own project in Windows or Linux use the c api:
+For example :
+we use the cmakefile to build project
+
+   IF (WIN32)
+      MESSAGE(STATUS "Now is windows")
+      add_definitions(-DUSE_STATIC_LIB -DTHREADED)
```

Comment:
Ideally, the `-DTHREADED` preprocessor symbol should only be used if the library was compiled in that mode.  But I suppose we could leave it in unconditionally as it does not cause any miscompilations, but that's not very "clean" as it still pushes errors from the compiling to the linking phase.

Reply:
`-DTHREADED` should also be defined in the non-Windows case, as the other fragment unconditionally links with `zookeeper_mt` (the `_mt` suffix meaning "multi-threaded"; the program should link to `zookeeper_st` otherwise).